spec/controllers/admin/admin_users_controller_spec.rb

```diff
@@ -463,14 +474,19 @@ def expect_changes_made_by(admin, log_items)
         end
       end
 
-      context "when user is banned" do
-        it "allows admins to destroy user creations" do
+      context "when deleting the first user" do
+        it "deletes the first user's collection but preserves the second user's collection" do
           admin.update(roles: ["policy_and_abuse"])
           fake_login_admin(admin)
           user.update(banned: true)
-          post :confirm_delete_user_creations, params: { id: user.login }
-
-          expect(response).to have_http_status(:success)
+          expect do
+            post :confirm_delete_user_creations, params: { id: user.login }  
+            post :destroy_user_creations, params: { id: user.login }
+          end.to change(Collection, :count).by(-1)
+          # Check that the first user's collection is deleted
+          expect(Collection.exists?(collection1.id)).to be_falsey
+          # Check that the second user's collection still exists
+          expect(Collection.exists?(other_owner.collections.last.id)).to be_truthy
```

Comment:
As we're effectively testing two different things, I feel like we want to keep the existing test as is, and make a fully new, additional test (even if it's 50% a copy-paste of the old one).